scripts/release_scripts/update_changelog_and_credits.py

```diff
@@ -43,14 +42,21 @@
 
 ABOUT_PAGE_FILEPATH = os.path.join(
```

Comment:
ABOUT_PAGE_**CONSTANTS**_FILEPATH, maybe? (I'm fine with this one as well but I found this constant name better for the html page (idkw))

Reply:
Done.